### PR TITLE
Cosets + minor changes

### PR DESCRIPTION
**Pull request by mazsa**
1. htmldefs, etc.
2. new symbols
3. the final version of the comments for ` ~ el*v* `
4. dfxrn2, but commented out, because one necessary theorem is in the Mathbox of @tirix
5. new section for the cosets-related theorems

### CODE REVIEW

**Comment by mazsa** [comment]
@tirix thank you for your https://github.com/metamath/set.mm/pull/2519 : I think I will use it only in my next PR in order to avoid complications with git:)

**Comment by benjub** [comment]
> @tirix thank you for your #2519 : I think I will use it only in my next PR in order to avoid complications with git:)

yes: doing this in another PR once this one is merged would be wise

**Comment by benjub** [comment]
Out of curiosity, @mazsa : was your second commit (minimization of six theorems) the result of `MM-PA>minimize_with *` ?

**Comment by mazsa** [comment]
> Out of curiosity, @mazsa : was your second commit (minimization of six theorems) the result of `MM-PA>minimize_with *` ?

@benjub yes. It was with ` minimize with * /include_mathboxes ` . It is in my regular workflow since you mentioned somewhere that it is expected even in Mathboxes.

File: set.mm
Comment (benjub) [inline set.mm:555511]: extra space before period

File: set.mm
Comment (benjub) [inline set.mm:555874]: Alternate definition of...
(same for previous statement)

**Review by benjub — APPROVED** [review]
Approved after cursory review since this is in a mathbox.

Your mathbox, so your choice to experiment with various definitions and variants, but I would recommend, when possible, to use existing definitions, like ~df-ec and related results in that section.

**Review by benjub — APPROVED** [review]

**Review by tirix — APPROVED** [review]

**Review by tirix — APPROVED** [review]

**Pull request merged**
